tutorials/synthetic-retrieval-evaluation/SDG-Retriever_Eval_Tutorial.ipynb

```diff
@@ -2897,18 +2918,20 @@
    ],
    "source": [
     "re_written_questions = []\n",
-    "        \n",
-    "def process_question(question):\n",
+    "\n",
+    "\n",
+    "def process_question(question: str) -> list[dict[str, str]]:\n",
     "    question_variants = []\n",
     "    for style in writing_styles:\n",
     "        try:\n",
-    "            re_write = generator.persona_rewrite(style['style'], question)\n",
+    "            re_write = generator.persona_rewrite(style[\"style\"], question)\n",
     "            question_variants.append({\"new_question\": re_write, \"style\": style, \"original_question\": question})\n",
-    "        except:\n",
-    "            continue\n",
-    "    print(\"Processed Question: \"+ question)\n",
+    "        except:  # noqa: E722, S110, PERF203\n",
+    "            pass\n",
+    "    print(\"Processed Question: \" + question)\n",
```

Comment:
Don't `continue` and `pass` change the logic here? The former breaks us out of the for loop while the latter continues the for loop.

Reply:
Reverted it back